distribution/changelog.txt

```diff
@@ -106,7 +107,7 @@
 - Feature: [#5611] Add support for Android
 - Feature: [#5706] Add support for OpenBSD
 - Feature: OpenRCT2 now starts up on the display it was last shown on.
-- Feature: Park entrance fee can now be set to amounts up to £200.
+- Feature: Park entrance fee can now be set to amounts up to �200.
```

Comment:
?

Reply:
Huh, must be a bug with the text editor I use.